docker.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -x
+RUST_TOOLCHAIN=1.49
```

Comment:
This version of Rust does not work with the aarch64 musl target. Can you leave it as 1.46?